www/src/pages/en/usage/env-variables.mdx

```diff
@@ -165,7 +171,11 @@ export const processEnv = {
 };
 ```
 
-_**NOTE:** An empty string is still a string, so `z.string()` will accept an empty string as a valid value. If you want to make sure that the environment variable is not empty, you can use `z.string().min(1)`._
+<Callout type="info">
+  An empty string is still a string, so `z.string()` will accept an empty string
```

Comment:
is single lines being split into multiple lines (here and elsewhere) a prettier thing?

Reply:
Yes, it is